apps/discord/commands.py

```
@@ -0,0 +1,108 @@
+"""Discord slash command definitions and handlers."""
+
+from typing import Dict, List
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)